.idea/readme.md

```diff
@@ -0,0 +1,121 @@
+<h2>Local Development & Docker Integration w/ Pycharm</h2>
+
+
+This document contains information regarding ways to use Lean’s Docker image in conjunction with local development in Pycharm.
+
+<h2>Getting Setup</h2>
+
+
+Before anything we need to ensure a few things have been done:
+
+
+
+1. Get [Pycharm Professional](https://www.jetbrains.com/pycharm/)**
+
+2. Get [Docker](https://docs.docker.com/get-docker/):
+    *   Follow the instructions for your Operating System
+    *   New to Docker? Try docker getting-started
+3. Pull Lean’s latest image from a terminal
+    *   _docker pull quantconnect/lean_
+
+     
+4. Clone LEAN Locally
+    *   git clone[ https://github.com/QuantConnect/Lean.git](https://github.com/QuantConnect/Lean.git)
+
+5. Open the folder in Pycharm
+
+**PyCharm’s remote debugger requires PyCharm Professional.
+
+<h2>Develop Algorithms Locally, Run in Container</h2>
+
+
+We have set up a relatively easy way to develop algorithms in your local IDE and push them into the container to be run.
+
+Before we can use this method with Windows or Mac OS we need to share the Lean directory with Docker.
+
+<h3>Activate File Sharing for Docker:</h3>
+
+
+
+
+    *   Windows
+        1. [https://docs.docker.com/docker-for-windows/#file-sharing](https://docs.docker.com/docker-for-windows/#file-sharing)
+        2. Share the LEAN root directory with docker
+    *   Mac
+        3. [https://docs.docker.com/docker-for-mac/#file-sharing](https://docs.docker.com/docker-for-mac/#file-sharing)
+        4. Share the LEAN root directory with docker
```

Comment:
I think 1 and 2 wanted to be a hyperlink? not sure if it's working for some reason
![image](https://user-images.githubusercontent.com/18473240/88927249-8857ed80-d24d-11ea-8de3-74e8bee12b27.png)

edit: confirmed, it's a hyperlink for vscode

Reply:
Fixed!